src/site/xdoc/server/config-cassandra.xml

```diff
@@ -38,6 +38,19 @@
         <dt><strong>cassandra.keyspace</strong></dt>
         <dd>Is the name of the keyspace used by James.</dd>
 
+        <dt><strong>cassandra.user</strong></dt>
+        <dd>Username used as a credential for contacting Cassandra cluster. Optional, default is absent,
+          required if <strong>cassandra.password</strong> is supplied</dd>
+
+        <dt><strong>cassandra.password</strong></dt>
+        <dd>Password used as a credential for contacting Cassandra cluster. Optional, default is absent,
+          required if <strong>cassandra.user</strong> is supplied</dd>
+
+        <dt><strong>cassandra.ssl</strong></dt>
+        <dd>Whether SSL should be enabled on the communications with Cassandra cluster. Optional, defaults to false.<br/>
+        The keystore used for trusting SSL server socket can be set via JSSE system properties as explained on
+        <a href="https://docs.datastax.com/en/developer/java-driver/3.7/manual/ssl/">Cassandra driver manual</a>.</dd>
+
         <dt><strong>cassandra.replication.factor</strong></dt>
         <dd>Is the replication factor. (should be 1, as cluster is not yet supported)</dd>
```

Comment:
Not related to your PR, but is it still true ?

Reply:
No, of course

Reply:
See https://github.com/linagora/james-project/pull/2800